src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+public class Storage {
+
+    protected String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Checks if the file in the given filePath exists.
+     * @return true if the file exists, or false otherwise.
+     */
+    public boolean isExistFile() {
+        return new java.io.File(filePath).exists();
+    }
+
+    // Idea for the following code snippet to create a file is referenced from:
+    // https://stackoverflow.com/questions/6142901/how-to-create-a-file-in-a-directory-in-java
+    /**
+     * Returns the desired File in the filePath.
+     * If there was originally no File, a new empty File with a "0" is created.
+     * Otherwise, the previously saved File at that filePath is returned.
+     * @return the desired File in the filePath.
+     */
+
+    public File load() {
+        // Idea for the following code snippet to create a file is referenced from:
+        // https://stackoverflow.com/questions/6142901/how-to-create-a-file-in-a-directory-in-java
+        if (!isExistFile()) {
+            Path desiredPath = Paths.get(filePath);
+            try {
+                Files.createDirectories(desiredPath.getParent());
+            } catch (IOException err) {
+                System.out.println(err);
+            }
+
+            try {
+                Files.createFile(desiredPath);
+            } catch (FileAlreadyExistsException err) {
+                System.err.println("already exists: " + err.getMessage());
+            } catch (IOException err) {
+                System.out.println(err);
+            }
+
+            try {
+                Files.write(Paths.get(filePath), "0".getBytes());
+            } catch (IOException err) {
+                System.out.println(err);
+            }
```

Comment:
Perhaps you could combine the 3 try-catch blocks into a single try-catch-catch block?